markdown/developers/review_checklist.md

```diff
@@ -0,0 +1,307 @@
+
+### Use the template
+
+All sanger-tol pipelines must be built using the nf-core template with a custom prefix `sanger-tol`.
+
+Workflows should be started using the `nf-core create` command which makes a new git repository and the initial commits and branches. This is to ensure that the sync process can work. See the [sync docs](https://nf-co.re/docs/contributing/sync) for details.
+
+Where possible, workflow authors should do their best to follow nf-core conventions for filenames and code locations.
+
+### Software license
+
+All sanger-tol pipelines must be released with an MIT license. The copyrights belong to Genome Research Ltd. as per Wellcome Sanger Institute policy.
+
+Please try not bundle any third party scripts within the workflow, in case they have a different or incompatible license (for example, in the `bin` directory). If you need such a script, even a simple one, please release it on bioconda instead and reference it like any other software.
+
+### Bundled documentation
+
+All documentation must be bundled with the pipeline code in the main repository, within a directory called `docs`.
+
+Documentation must _only_ be hosted on the pipelines website and GitHub.
+Hosting the documentation at a second location (such as custom readthedocs website, or GitHub pages etc) is not allowed.
+This is to ensure that users of sanger-tol pipelines can always intuitively find the documentation for all sanger-tol pipelines in the same way.
+
+Documentation must include at least the following files:
+
+- `README.md`
+- `docs/usage.md`
+- `docs/output.md`
+
+Additional pages (e.g. tutorials, FAQs) can be added and will be automatically rendered on the website pipeline page.
+
+### Docker support
+
+Pipelines must have all software bundled using [Docker](https://www.docker.com/) - that is, it must be possible to run the pipeline with `-profile docker` and have all software requirements satisfied.
+
+Tools should use docker images from [biocontainers](https://biocontainers.pro) where possible, as using Bioconda / Biocontainers gives support for conda + docker + singularity.
+
+All containers must have specific, stable versions pinned.
+These should preferably be named after a software release, but it can also be by commit or some other identifier.
+Software versions must be static and stable. Labels such as `latest`, `dev`, `master` and so on are not reproducible over time and so not allowed.
+
+### Continuous integration testing
+
+Pipelines must have automated continuous integration testing, running using GitHub Actions. There must also be CI tests using Nextflow Tower (`sanger_test` and `sanger_test_full`).
+
+There must be a config `profile` called `test` that should be as comprehensive as possible - that is, it should run as much of the pipeline as possible. It should use as tiny test data set as possible (even if the output that it creates is meaningless).
+
```

Comment:
`cleanup` profile is not the compulsory condition for the Github action. Can we move adding this profile to somewhere else?

The purpose of running tests via Tower is to submit the job to Sanger farm, can we include that here?

Reply:
I've modified the two suggestions above accordingly. Yes, the `cleanup` profile is not mandatory (and I don't say it is !) but it's part of our SOP for getting the Tower/LSF integration to run. It's the only reason why we have this profile, as otherwise we would use `-cleanup true` on the command-line.